transmart-core-db/src/main/groovy/org/transmartproject/db/multidimquery/query/Constraint.groovy

```diff
@@ -281,6 +283,12 @@ class FieldConstraint extends Constraint {
     Object value
 
     static constraints = {
+        field validator: { val, obj, Errors errors ->
```

Comment:
I think that [@BindUsing({ obj, source -> ConstraintFactory.bindField(obj, 'field', source['field']) })](https://github.com/thehyve/transmart-core/blob/8e6cba6583599dd474c09c39e77df594d9a3cf17/transmart-core-db/src/main/groovy/org/transmartproject/db/multidimquery/query/Constraint.groovy#L279) is already checking if field is set, but I am not 100% sure.